### DISCUSSION TITLE
Linux基金会企业开源指南系列之三--一份让你彻底理解开源的读书列表 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#14] [Announcements]
# Linux基金会企业开源指南系列之三--一份让你彻底理解开源的读书列表 - 开源之道

https://opensourceway.community/posts/opensource_enterprise_guide/open_source_reading_list/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
作者不是故意选择英文为主的图书,
而是相反, 在中文世界, 被翻译过的好书,
基本上都是 程序员 杂志还生存的时代,
随着移动互联网时代的到来, 云 基本替代了商业软件市场空间,
中国也有了一批极大体量的巨头IT公司,
将民间程序猿都吸走了,
而中国的开源社区本质上进入了热寂期:
+ BLUG 一直都是外国在华工程师俱乐部, 没有什么变化, 对中国开发者影响也小
+ SHLUG 之类早期社区, 老人逐一出海, 新人不知道如何进入, 变成在地小众私人"会所"
+ ...各种新创建的原创开源产品社区, 也都老实进入 Apache 基金会管理赛道, 接受整改, 也没有溢出高端人群

所以, 近期结合经济/社会/人类/心理/学习科学/...学科的开源研究/思考图书, 根本没有翻译动力...

所以, 就看原文吧. 反正有 google

**Reply by ZoomQuiet** [reply]
PS:
忘记说了, 其实, 以上图书你可以一本都不看, 也可以真实理解 FLOSS 世界,
而且更加高效:

投身到这一运动中, follow u heart 创造出全新作品, 并吸引到足够多的伙伴,
一切自在心中.

**Reply by lijiangsheng1** [reply]
选择本身就是难题，我遇到的年轻人，不是像我当年那样开源是唯一的选择，而是年轻人们选择太多：开源毫无吸引力～ 他们从来不在乎自由！ 早就被规训了。
